README.md

```diff
@@ -81,6 +81,7 @@ This list should be reviewed and pruned annually (at minimum). The calendar has
 - [@bethgriggs](https://github.com/bethgriggs) - **Beth Griggs**
 - [@bnb](https://github.com/bnb) - **Tierney Cyren**
 - [@gibfahn](https://github.com/gibfahn) - **Gibson Fahnestock**
```

Comment:
Unrelated to this PR except perhaps indirectly, but I don't think @gibfahn has been active for quite some time and we should probably remove their access to the calendar in the name of good security hygiene.

Reply:
Submitted a PR.